package.json

```diff
@@ -108,6 +127,7 @@
   },
   "homepage": "https://buttercup.pw",
   "dependencies": {
+    "assert": "^1.4.1",
```

Comment:
Can you move this to `devDependencies` please?

Reply:
jep